acc_test/resource_cluster_test.go

```diff
@@ -680,3 +680,71 @@ func testCheckResourceStateAttributesDeleted(resourceName string, attributeNames
 		return nil
 	}
 }
+
+func TestCheckNoDiffOldToNewBundleVersionFormat(t *testing.T) {
```

Comment:
I'm not keen to add more acc_tests that create more clusters. It slows everything down.
Is there a valid way to unit test this? Or tp piggyback it off a cluster that has already been created as part of these tests?

Reply:
Yeah I should be able to attach to an existing provisioning test. I'll refactor it to that. As for unit testing, I'll take a look into it.